.github/workflows/shippable_builds.yml

```diff
@@ -204,17 +205,25 @@ jobs:
 
       - name: Render Release Notes
         if: ${{ contains(matrix.releaseTarget, 'github') }}
+        id: render_notes
         shell: bash
         env:
+          APP_NAME: ${{ matrix.appName }}
           APPLICATION_ID: ${{ steps.appinfo.outputs.APPLICATION_ID }}
           APPLICATION_LABEL: ${{ steps.appinfo.outputs.APPLICATION_LABEL }}
           VERSION_CODE: ${{ steps.bump_version_code.outputs.CODE }}
           FULL_VERSION_NAME: ${{ steps.appinfo.outputs.VERSION_NAME }}${{ steps.bump_version_suffix.outputs.SUFFIX || steps.appinfo.outputs.VERSION_NAME_SUFFIX }}
         run: |
           echo "<h2>${APPLICATION_LABEL} ${FULL_VERSION_NAME} Release Notes (${VERSION_CODE})</h2><pre>" | tee -a $GITHUB_STEP_SUMMARY
           mkdir -p ./app-metadata/${APPLICATION_ID}/en-US/changelogs
-          python ./scripts/render-notes.py ${APPLICATION_ID} ${FULL_VERSION_NAME} ${VERSION_CODE} | tee -a $GITHUB_STEP_SUMMARY
+          GITHUB_NOTES_FILE="$(mktemp -d)/long-notes.txt"
+          python ./scripts/render-notes.py -p changelog -- ${APPLICATION_ID} ${FULL_VERSION_NAME} ${VERSION_CODE} ${GITHUB_NOTES_FILE} | tee -a $GITHUB_STEP_SUMMARY
           echo "</pre>" | tee -a $GITHUB_STEP_SUMMARY
+          if [[ "$APP_NAME" == "thunderbird" ]]; then
+            echo "thunderbird_github_notes<<EOF" >> $GITHUB_OUTPUT
+            cat $GITHUB_NOTES_FILE >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          fi
```

Comment:
Both K-9 Mail and Thunderbird for Android have GitHub Releases, so they both have long form notes. There are a few items that will be in the Thunderbird release notes that won't be in the K-9 Mail release notes, I don't think we talked about how to differentiate in the notes yaml.